frontend/tests/features/cellGuide/cellGuide.test.ts

```diff
@@ -398,6 +398,52 @@ describe("Cell Guide", () => {
           { page }
         );
       });
+      test("Enriched marker gene table is updated by the organ dropdown", async ({
+        page,
+      }) => {
+        await goToPage(
+          `${TEST_URL}${ROUTES.CELL_GUIDE}/${T_CELL_CELL_TYPE_ID}`,
+          page
+        );
+
+        await tryUntil(
+          async () => {
+            // set enriched marker genes table as active
+            await page
+              .getByTestId(CELL_GUIDE_CARD_ENRICHED_GENES_TABLE_SELECTOR)
+              .click();
+
+            const tableSelector = `[data-testid='${CELL_GUIDE_CARD_ENRICHED_GENES_TABLE}']`;
+            await page.locator(tableSelector).waitFor({ timeout: 5000 });
+
+            const rowElementsBefore = await page
+              .locator(`${tableSelector} tbody tr`)
+              .elementHandles();
+            const rowCountBefore = rowElementsBefore.length;
+            expect(rowCountBefore).toBeGreaterThan(1);
+            const firstRowContentBefore =
+              await rowElementsBefore[0].textContent();
+
+            const dropdown = page.getByTestId(
+              CELL_GUIDE_CARD_MARKER_GENES_TABLE_DROPDOWN_ORGAN
+            );
+            await waitForElementAndClick(dropdown);
+            await dropdown.press("ArrowDown");
+            await dropdown.press("ArrowDown");
+            await dropdown.press("Enter");
+
+            const rowElementsAfter = await page
+              .locator(`${tableSelector} tbody tr`)
+              .elementHandles();
```

Comment:
let's deprecate the usage of `elementHandles()` as much as possible! In this case, I think we can use `all()`?

<img width="787" alt="Screenshot 2023-08-09 at 8 21 33 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/33f2ef7f-d981-4c95-bd16-c3ba6f821637">